validator/src/commands/run/args.rs

```diff
@@ -1710,8 +1713,12 @@ mod tests {
     use {
         super::*,
         crate::cli::thread_args::thread_args,
+        solana_rayon_threadlimit::get_thread_count,
```

Comment:
I'm looking to kill this crate, but the usage here matches existing behavior so let's proceed with it and I'll clean this up after this PR ships